internal/commands/document.go

```diff
@@ -89,7 +100,11 @@ func runDocCommand(path string) error {
 		return fmt.Errorf("opening file for writing: %w", err)
 	}
 
-	if err := t.Execute(f, docs); err != nil {
+	tmplData := DocsMeta{
+		Documentation: docs,
+		IncludeRego:   !viper.GetBool("no-rego"),
```

Comment:
thought: I think it may be cleaner to just not set the `Rego` property inside of our `Document` type when this field is set. Then our documentation template can just conditionally render the Rego or not _if_ Rego is not blank.

Reply:
Yeah, that would be cleaner. Updated.